src/components/Error/Error.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  errorMessage: string,
+  removeMessage: () => void,
+};
+
+export const Error: React.FC<Props> = (
+  { errorMessage, removeMessage },
+) => {
+  const isError = errorMessage !== '';
```

Comment:
there should be the case such `errorMessage === ''` because you should render the Error compnent at all if there is no errorMessage